api/api/models/media.py

```diff
@@ -275,14 +275,15 @@ def _perform_index_update(self, method: str, raise_errors: bool, **es_method_arg
                     refresh=True,
                     **es_method_args,
                 )
-            except TransportError as e:
-                if e.status_code == 404:
+            except (NotFoundError, BadRequestError) as e:
+                if isinstance(e, NotFoundError):
```

Comment:
Why not two separate except clauses for `NotFoundError` and `BadRequestError`? Also if we are raising the latter again, why do we need to catch it at all?

Reply:
Totally! I only realised this while annotating the changes just now. Will remove the BadRequestError here.

Reply:
Cleaned up in https://github.com/WordPress/openverse/pull/3029/commits/fe96cb145ddd7ea8548805411b208dfd8ab47c7b